App.tsx

```diff
@@ -29,12 +30,8 @@ const bgImage = require('./assets/bg-default.jpg');
 
 export default function App() {
 
-  //get default image details
-  const defaultSource = RnImage.resolveAssetSource(bgImage);
-  const { uri, width, height } = defaultSource;
-
   //various variables
-  const [background, setBackground] = useState<ImageTypes>({ uri, width, height });
+  const [background, setBackground] = useState(bgImage);
```

Comment:
Passing the result of `require(assets)` to `ImageMarker` instead of URI only otherwise Coil will use the resolution of the current device to set the width and height of the image

Reply:
Great, thanks. This would still leave an issue with instances of ImagePickerResult and ImageResult (returned by ImageManipulator.manipulateAsync()), correct?

Reply:
> Great, thanks. This would still leave an issue with instances of ImagePickerResult and ImageResult (returned by ImageManipulator.manipulateAsync()), correct?

Yes, it is. I will fix it by specifying that Coil should use the original dimensions when loading images without a set size.

Reply:
👍